cypress/support/helper-functions/participant/participant-helpers.js

```diff
@@ -0,0 +1,22 @@
+export const timeout = Cypress.env('timeout');
+
+export function signInWithClever() {
+  cy.wait(0.2 * timeout);
+  cy.get('button', { timeout: timeout }).contains('Clever').click();
+
+  cy.origin('https://clever.com/oauth/authorize', () => {
```

Comment:
I would save this URL as an env variable in the Cypress config file.

Reply:
Good point, will do.